imagery/i.atcorr/main.cpp

```diff
@@ -204,12 +204,10 @@ class TICache {
 private:
     struct RBitem set_alt_vis(double alt, double vis)
     {
-        struct RBitem rbitem;
-
+        struct RBitem rbitem = {};
         /* alt and vis must be in meters */
         rbitem.alt = (alt < 0 ? (int)(alt - 0.5) : (int)(alt + 0.5));
         rbitem.vis = (int)(vis + 0.5);
-
```

Comment:
Please revert this unrelated change.

Reply:
i added the empty line back which was deleted.

Reply:
reverting back and adding the empty line causes Clang formatting issues.

Reply:
> reverting back and adding the empty line causes Clang formatting issues.

It’s because of trailing whitespaces.

Reply:
Is doing that change necessary.

Reply:
Not functionally, but stylistically. To be able to merge this, it has to pass all required tests.

Reply:
We’re back where we started with this, deleting an unrelated empty line. To revert this, add an empty line with no spaces (!) before the return line in a new commit

Reply:
Done